assets/templates/login.html

```diff
@@ -1,46 +1,68 @@
-<!DOCTYPE html>
+  <!DOCTYPE html>
```

Comment:
Can we remove the spaces before the doctype? I'm not sure but I think that the doctype has to start from the first byte to be well supported